samples/bot-proactive-messaging-teamsfx/bot/teamsBot.js

```diff
@@ -3,6 +3,11 @@
 
 const { TeamsActivityHandler, TurnContext } = require("botbuilder");
 
+const path = require('path');
+
+// Read botFilePath and botFileSecret from .env file.
+require('dotenv').config({ path: '../env/.env.local' }); // If deploying or provisioning the sample, please replace this with with .env.dev
```

Comment:
Please mention this in the Readme file:
// If deploying or provisioning the sample, please replace this with with .env.dev

Reply:
added